### ISSUE TITLE
PDF documents are not accessible

### ISSUE THREAD

**Issue by khays9** [UKHomeOffice/posters#62]
The PDF documents are not accessible (though you do have a text-only alternative). Consider tagging the PDFs for accessibility.

**Comment by karypun** [comment]
@khays9 can you explain this in more detail please. We've reviewed the pdfs and made sure they were formatted correctly, but if we missed anything, would be good to pinpoint exactly what it is that makes it inaccessible. Thank you.

**Comment by csogi** [comment]
You can run an accessibility check on the pdf via Acrobat DC. 

Here is a report and the common mistakes. 
<img width="327" alt="adatestreport" src="https://user-images.githubusercontent.com/6052006/31840655-50ad39a2-b5b4-11e7-92ee-3c6a2de7eef4.png">

[AcrobatAccessibilityReport.pdf](https://github.com/UKHomeOffice/posters/files/1403207/AcrobatAccessibilityReport.pdf)

How to run check in acrobat https://helpx.adobe.com/acrobat/using/create-verify-pdf-accessibility.html

**Comment by khays9** [comment]
My apologies for not being specific. One example, is that the en-UK version of the autistic-spectrum.pdf poster has multiple issues, including lack of tagged content and alternate text for figures. Other files are tagged accessibly but have issues, such as the en-UK version of the accessibility-posters-set.pdf file, where headings are not nested appropriately (as csogi posted above). It would be good to run the Acrobat accessibility report for all files and correct any issues found.

Issues like no title or improper nesting for headings are relatively minor compared to lack of accessible tagging or alternate text. Reading order is another issue that usually requires a manual check. Adobe's documentation on making pdfs accessible is very good. For a more concise tutorial, you can look at the [https://itaccessibility.illinois.edu/pdf](IT Accessibility at Illinois PDF section).

**Comment by karypun** [comment]
@csogi @khays9 thank you for pointing to the accessibility checking feature in Acrobat and for the specific advice. Will take this all on board and have our team review as well.

**Comment by jdblundell** [comment]
I noticed the reading order was off on several of the English versions of the posters and the links in the footer were not properly tagged. Also the Creative Commons language appears to be an image that's embedded in the page somehow so it can't be tagged.
I'm having issues adding this to GitHub so here's my updated version if it helps.
[accessibility-posters-set.pdf](https://github.com/UKHomeOffice/posters/files/3220036/accessibility-posters-set.pdf)

**Comment by SueSpevack** [comment]
@jdblundell thanks for getting in touch in regards to this and making us aware of these issues. Could I ask which screen reader and device you used to check the issues you have mentioned?

**Comment by jdblundell** [comment]
thanks. It’s actually been a little while since I submitted the issue so I’m not 100% but I typically test with JAWS or review the tags in the PDF.
    	

    	Jonathan Blundell@jdblundell
    
  



On Tue, Jun 11, 2019 at 9:11 AM -0500, "Sue Spevack" <notifications@github.com> wrote:












@jdblundell thanks for getting in touch in regards to this and making us aware of these issues. Could I ask which screen reader and device you used to check the issues you have mentioned?
